package.json

```
@@ -29,8 +29,8 @@
     "@babel/cli": "^7.5.5",
     "@babel/core": "^7.5.5",
     "@babel/preset-env": "^7.15.0",
-    "@types/node": "^20.4.9",
     "@types/jest": "latest",
+    "@types/node": "^20.4.9",
```

Node version and `@types/node` major and minor version should match. Latest node LTS is `18.17`
```suggestion
    "@types/node": "^18.17.9",
```